src/runtime/virtcontainers/sandbox.go

```diff
@@ -638,6 +642,23 @@ func newSandbox(ctx context.Context, sandboxConfig SandboxConfig, factory Factor
 	return s, nil
 }
 
+func setHypervisorConfigAnnotations(sandboxConfig *SandboxConfig) {
+	if len(sandboxConfig.Containers) > 0 {
+		// These values are required by remote hypervisor
+		for _, a := range []string{cri.SandboxName, crio.SandboxName} {
```

Comment:
It took me 5 min looking at this line to realize the "cri" and "crio" tokens! hehehe I was like "why the entries are equal?" /me getting old....